src/aws/ffmpeg-transmux-live-image/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
Maybe you want to add a `set -e` to fail if one of the command fails ?
Or do you want to invoke the lambda even in case of failure ?

Reply:
No I will add the `set -e` at the beginning